core/src/main/java/jenkins/security/s2m/ConfigFile.java

```diff
@@ -26,15 +27,17 @@ public ConfigFile(File file) {
     protected abstract COL create();
     protected abstract COL readOnly(COL base);
 
-    public synchronized void load() {
+    public synchronized void load() throws IOException {
```

Comment:
It breaks the source compatibility IIUC. I would rather introduce a new one and deprecate `load()`

Reply:
Why do we care about source compatibility?

Reply:
PCT?